cmd/soroban-cli/src/commands/contract/extend.rs

```diff
@@ -84,6 +84,8 @@ pub enum Error {
     Data(#[from] data::Error),
     #[error(transparent)]
     Network(#[from] network::Error),
+    #[error(transparent)]
```

Comment:
Is this needed? Doesn't it come from the config::Error?

Reply:
not really! The locator module has its own `Error` enum. See https://github.com/stellar/stellar-cli/blob/41648d99d1714fa1ea0d3788a11508ebcea7904a/cmd/soroban-cli/src/commands/config/locator.rs#L19

Reply:
Yeah but it's added to the config::Error so it will be caught. The quick way to check is if the compiler is made. Either way I think there are Errors all over that aren't needed so a future PR could clean them up but it's not a big issue.